scoper.inc.php

```diff
@@ -43,6 +43,7 @@
         'Automattic',
         'Composer',
         'MyParcelNL',
+        'MyParcelBE',
```

Comment:
Hoe kan dit? Er is niks wat begint met MyParcelBE en de scoper wordt gerund voordat de resultaten worden getransformeerd naar andere platformen